frontend/public/components/utils/section-heading.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+export const SectionHeading: React.SFC<SectionHeadingProps> = ({text, children, style}) => <h2 className="co-section-heading" style={style}>{text}{children}</h2>;
```

Comment:
Curious if the `text` property is necessary, or if we can treat `<SectionHeading />` just like an `<h1>` `<h2`, etc:

```html 
<h1>A heading</h1>
<SectionHeading>Another!</SectionHeading>
<h1>A heading <small>with something else</small></h1>
<SectionHeading>A heading <small>anything quirky gonna happen?</small></SectionHeading>
<SectionHeading text="A heading"><small>this works just like the prev?</small></SectionHeading>
```
As log as both works, we prob fine!

Reply:
@benjaminapetersen Agree this is probably better. @sg00dwin is reusing an existing component and simply pulled it out into a separate file. Wondering if this is worth changing at this point.